probe/endpoint/connection_tracker.go

```diff
@@ -87,7 +91,27 @@ func (t *connectionTracker) ReportConnections(rpt *report.Report) {
 			t.performEbpfTrack(rpt, hostNodeID)
 			return
 		}
-		log.Warnf("ebpf tracker died, gently falling back to proc scanning")
+
+		// We only restart the EbpfTracker if the failures are not too frequent to
+		// avoid repeatitive restarts.
+		if t.ebpfFailureCount > 0 && t.ebpfFailureTime.Before(time.Now().Add(-5*time.Minute)) {
+			t.ebpfFailureCount = 0
+		}
+		t.ebpfFailureTime = time.Now()
+
+		if t.ebpfFailureCount == 0 {
+			log.Warnf("ebpf tracker died, restarting it")
+			t.ebpfFailureCount++
+			err := t.ebpfTracker.restart()
+			if err == nil {
+				go t.getInitialState()
```

Comment:
Shouldn't the invocation of `getInitialState()` be part of the tracker initialization in `restart()`?